src/components/RepoList/RepoList.module.scss

```
@@ -0,0 +1,8 @@
+.repo-list {
+  width: 100%;
+  padding: 0;
+  margin: 0;
+  margin-top: 20px;
```

스타일 중첩의 경우 순서가 바뀌거나, 추후 유지보수 할 때 어려움이 있을 듯 합니다.
아래와 같이 사용하는 것은 어떤가요?

```suggestion
  margin: 20px 0 0;
```

---

아 말씀해주신 방법이 더 깔끔하고 좋을꺼같아요! 참고하겠습니다!